support/bazel/README.md

```diff
@@ -0,0 +1,110 @@
+# Bazel support
+
+## Bazelize fmt
+
+To get [Bazel](https://bazel.build/) working with `fmt` you can copy the files `BUILD.bazel`, `WORKSPACE.bazel`, `.bazelrc`, and `.bazelversion` from this folder (`support/bazel`) to the root folder of this project.
+
+This way `fmt` gets bazelized and can be used in your Bazel builds.
+
+**Example**
+
+Create a `WORKSPACE.bazel` file with the following content:
+
+```python
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+# Fetch fmt
+git_repository(
+    name = "fmt",
+    branch = "bazel-support", # A copy of master where BUILD.bazel, WORKSPACE.bazel, .bazelrc and .bazelversion are move to root
+    remote = "https://github.com/<user_or_organistzion>/fmt", # replace <user_or_organisation> by a valid accunt
+)
+```
+
+Create a `BUILD.bazel `file and add a dependency to `fmt`:
+
+```python
+cc_binary( # Build a binary
+    name = "Demo", # Name of the Binary
+    srcs = ["main.cpp"], # List of files - we only have main.cpp
+    deps = ["@fmt//:fmt"], # Depend on fmt
+)
+```
+
+Make use of `fmt` in `main.cpp`:
+
+```C++
+#include "fmt/core.h"
+
+#include <iostream>
+
+int main() {
+    std::string message = fmt::format("The answer is {}.", 42);
+
+    std::cout << message << std::endl;
+    
+    return 0;
+}
```

Comment:
Here and below I recommend using `fmt::print` instead of iostreams:

```c++
#include "fmt/core.h"

int main() {
  fmt::print("The answer is {}.\n", 42);
}
```

Reply:
Copied your example